GiniVision/Classes/Core/Screens/Document picker/DocumentPickerCoordinator.swift

```diff
@@ -225,6 +244,64 @@ extension DocumentPickerCoordinator {
         
         return nil
     }
+    
+    @available(iOS 11.0, *)
+    func saveCurrentAppAppearance() {
```

Comment:
It's not really the "App" appearance, is it? It's more like the navigation bar. I find the name misleading.